executor/python/mesh.py

```diff
@@ -0,0 +1,11 @@
+import grpc
+
+import fxwatcher_pb2
+import fxwatcher_pb2_grpc
+
+def mesh_call(functionName: str, input):
+    channel = grpc.insecure_channel("{}.openfx-fn:50052".format(functionName))
+    clientStub = fxwatcher_pb2_grpc.FxWatcherStub(channel)
+    serviceRequest = fxwatcher_pb2.Request(input = input, info = fxwatcher_pb2.Info(FunctionName = functionName, Trigger = fxwatcher_pb2.Trigger(Name = "grpc")))
```

Comment:
Google Style Guide나 PEP8 기준으로
python 변수명은 snake-case입니다!

`clientStub` -> `client_stub`
`serviceRequest` -> `service_request`

실은 gRPC 예제에서 CamelCase로 써서 난감하긴 합니다만..... ㅎㅎ

```suggestion
    client_stub = fxwatcher_pb2_grpc.FxWatcherStub(channel)
    service_request = fxwatcher_pb2.Request(input = input, info = fxwatcher_pb2.Info(FunctionName = functionName, Trigger = fxwatcher_pb2.Trigger(Name = "grpc")))
```